src/resolvers/projectResolver.ts

```diff
@@ -797,6 +828,20 @@ export class ProjectResolver {
     });
 
     const newProject = await this.projectRepository.save(project);
+    const adminUser = (await findUserById(Number(newProject.admin))) as User;
+    newProject.adminUser = adminUser;
+    for (const relatedAddress of projectInput?.addresses) {
```

Comment:
We have a N+1 queries here as we are saving entities separately. It's better to unify the jsons and insert once into the entity.

Reply:
@CarlosQ96 Thanks for mentioning that, I fixed it here
https://github.com/Giveth/impact-graph/pull/531/commits/6fc6f71cac105fa28293b135418a3b9e1ee63517